### ISSUE TITLE
Fail to converge when adapt the feature extractor backbone from efficientnet to R18 or R50

### ISSUE THREAD

**Issue by alvinliu0** [autonomousvision/projected_gan#72]
Hi, this is a really impressive and exciting work!

When I try to adapt the feature extractor backbone from efficientnet to R18 or R50, I find it hard for the network to converge. 
I follow your step to load timm pretrained resnet18 and resnet50, then define the 4-level feature hierarchy similarly. However, it seems that the FID is always 300+ even after 200 ticks. Is there any configuration or specific setting that I missed? Many thanks!

Below is my code for adapting feature extraction backbone. Other part of the code remains unchanged:

``` python
model = timm.create_model('resnet18', pretrained=True)
pretrained = _make_resnet18(model)

def _make_resnet18(model):
    pretrained = nn.Module()
    pretrained.layer0 = nn.Sequential(model.conv1, model.bn1, model.act1, model.maxpool, *model.layer1)
    pretrained.layer1 = nn.Sequential(*model.layer2)
    pretrained.layer2 = nn.Sequential(*model.layer3)
    pretrained.layer3 = nn.Sequential(*model.layer4)
    return pretrained

**Comment by alvinliu0** [comment]
Besides, could you please kindly share the code or the detailed setting for adapt the backbone to DeiT or ViT. 

E.g., which specific timm model do you use? Since in ViT or DeiT, all of H, W, and C dimensionality remains unchanged, do you simply remove the upsample operation in csm feature fusion block?

Many thanks again!

**Comment by sunkicheon** [comment]
I'm confronting the similar issue! I attempted using R50 with timm but it failed by simply replacing the feature network. I also tried increasing cout to 256 (with expand option = true), since R50 has much larger channel dimension size, i.e., 256, 512, 1024, 2048. While the FID score has been slightly improved but Recall remain 0, indicating possible mode collapse. It would be much helpful if important configurations or tips for employing Resnet based architectures can be shared.

I attach the training logs for your information. Thanks!

<img width="1031" alt="projected_gan_R50" src="https://user-images.githubusercontent.com/37164552/160537625-6a0d0c81-f980-467b-98ec-85bc28741d20.png">
